libs/shared/devops-copilot/feature/src/lib/devops-copilot-markdown/devops-copilot-markdown.tsx

```diff
@@ -0,0 +1,130 @@
+import { isValidElement } from 'react'
+import type { FC, HTMLAttributes } from 'react'
+import Markdown from 'react-markdown'
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter'
+import { materialDark } from 'react-syntax-highlighter/dist/esm/styles/prism'
+import remarkGfm from 'remark-gfm'
+import { CopyToClipboardButtonIcon } from '@qovery/shared/ui'
+import { MermaidChart } from '../mermaid-chart/mermaid-chart'
+
+type CodeProps = {
+  inline?: boolean
+  className?: string
+  children?: React.ReactNode
```

Comment:
Use `PropsWithChildren` from React type

```suggestion
  } extends PropsWithChildren
```